tests/Oryx.Tests.Common/ImageTestHelper.cs

```diff
@@ -517,8 +527,10 @@ public static class ImageTestHelperConstants
         public const string PackRepository = "pack";
         public const string CliRepository = "cli";
         public const string CliBusterRepository = "cli-buster";
+        public const string CliBullseyeRepository = "cli-bullseye";
```

Comment:
CliBusterRepository and CliBullseyeRepository are referenced in multiple cli tests to get the appropriate cli image name using the GetCliImage method like in [this](https://github.com/microsoft/Oryx/blob/main/tests/Oryx.BuildImage.Tests/Python/PythonDynamicInstallationTest.cs#L59)example.

However, since we would only have the cli repository now, it makes sense to remove CliBusterRepository and CliBullseyeRepository from here but we would have to make changes to all the tests calling this method(has 80+ references). 
If we decide to remove these constants, we would have to modify this method to check for the tag instead the repository which would lead to modifying all those 80+ references. 
So, I would think it would be okay to leave these as-is for the sake of simplicity. Any thoughts?

Reply:
@snehapar9 Its quite easy to rename a variable and all references in visual studio by selecting something like `CliBullseyeRepository` and hitting `ctrl + R + R`. I'd probably handle this for bullseye (and other flavors) by:
- deleting current `CliBullseyeTag` 
- `ctrl + R + R` renaming `CliBullseyeRepository` to `CliBullseyeTag`
- replacing the value `cli-bullseye` with `debian-bullseye`
- remove the `_cliBullseyeRepository` private var in this file
- fix the `GetCliImage` and `GetBuildImage` to use the cli bullseye tag

All in all, it will bloat this pr a bit for us reviewers now, but it shouldn't take too long and will make more sense for us in the future IMO

Reply:
By replacing cli-bullseye by debian-bullseye, you mean the arg we pass to the buildBuildImages.sh script like [here](https://github.com/microsoft/Oryx/blob/snehapar/Install-deps-dynamically/build/buildBuildImages.sh#L546)?
And you probably mean [these](https://github.com/microsoft/Oryx/blob/snehapar/Install-deps-dynamically/tests/Oryx.BuildImage.Tests/Node/NodeDynamicInstallationTest.cs#L101) traits tied to [these](https://github.com/microsoft/Oryx/blob/snehapar/Install-deps-dynamically/vsts/pipelines/validation.yml#L38) as well?

Reply:
> By replacing cli-bullseye by debian-bullseye, you mean the arg we pass to the buildBuildImages.sh script like [here](https://github.com/microsoft/Oryx/blob/snehapar/Install-deps-dynamically/build/buildBuildImages.sh#L546)?

@snehapar9 sorry no, was only referring to the the value of `CliBullseyeRepository`, which is currently `cli-bullseye`. 

> And you probably mean [these](https://github.com/microsoft/Oryx/blob/snehapar/Install-deps-dynamically/tests/Oryx.BuildImage.Tests/Node/NodeDynamicInstallationTest.cs#L101) traits tied to [these](https://github.com/microsoft/Oryx/blob/snehapar/Install-deps-dynamically/vsts/pipelines/validation.yml#L38) as well?

I'm not sure that those even need to be switched, as they don't refer to the CLI repository. They are just categories of tests, and `cli-bullseye` as a testing category still makes sense, regardless of which repo the image ends up in.

Reply:
Okay. That makes sense.